src/main/java/racingcar/view/OutputView.java

```diff
@@ -1,7 +1,33 @@
 package racingcar.view;
 
+import racingcar.domain.car.CarName;
+import racingcar.domain.car.Cars;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static racingcar.view.Marks.COMMA;
+
 public class OutputView {
+    public static final String EXECUTION_RESULT = "실행 결과";
+    public static final String MESSAGE_GAME_RESULT = "가 최종 우승했습니다.";
+
     public static void print(String message) {
         System.out.println(message);
     }
+
+    public static void printCarStates(Cars cars) {
+        cars.getCarList().stream().forEach(car -> {
```

Comment:
```suggestion
        cars.getCarList().forEach(car -> System.out.println(// 수정
```
이 부분이 위에서 말씀드린 도메인의 출력로직이 들어간 부분입니다!